Components/Pages/Login.js

```diff
@@ -1,18 +1,44 @@
-import * as React from "react";
-import LoginTemplate from "../Templates/LoginTemplate";
+import React from "react";
+import SignUpContainer from "../Molecules/SignUpContainer";
+import { TextField } from "../Atoms/Fields";
 
-class Login extends React.Component {
-  back = () => {
-    this.props.navigation.navigate("Welcome");
-  };
-
-  login = () => {
-    this.props.navigation.navigate("Home");
-  };
-
-  render() {
-    return <LoginTemplate back={this.back} login={this.login} />;
-  }
-}
+const Login = props => {
+  const { navigation, setPasswordRef, goToPassword } = props;
+  /* 
+    What are setPasswordRef, goToPassword? 
```

Comment:
I'm not sure what setPasswordRef and goToPassword Ref are since we inheritted that from the template. It has something to do with the TextInput from react-native